www/src/pages/404.astro

```diff
@@ -0,0 +1,25 @@
+---
+import BaseHead from '../components/BaseHead.astro';
+import Main from '../components/Main.astro';
+import MainHeader from '../components/MainHeader.astro';
+import Space from '../components/Space.astro';
+import ErrorPage from '../components/ErrorPage.astro';
+
+let title = 'Astro | Page Not Found';
+let description = 'Build faster websites with less client-side JavaScript';
+let permalink = 'https://astro.build/404';
+let lang = 'en';
+---
+<html lang={ lang ?? 'en' }>
```

Comment:
Is this necessary? I was just looking, the `lang` is already being hardcoded in the frontmatter, could you not just set the attribute, `<html lang='en'/>`

Reply:
Yeah, noticed that too, but it's done in every page, so I decided to follow. I also noticed that in `<BaseHead />` the `permalink` property should be renamed to `canonicalURL`